### ISSUE TITLE
some necessary files missed

### ISSUE THREAD

**Issue by xinli2008** [fabrahman/Emo-Aware-Storytelling#6]
Hello,i am trying to reproduce your project.
When i am trying to do run_emorl.sh , the file "x4_emo_features.dev" missed.
Can you give me some useful advice? @fabrahman

**Comment by fabrahman** [comment]
@shaoniana1997, `x4_emo_features.dev` and `x4_emo_features.train` files are the emotion probabilities for each emotion , and each gold story segment (beginning, body, end). I computed them upfront using the pre-trained classifier. They are required for a RL training with a different reward, `clf`, and not `clf_prob` which is used in the paper.
That is for the results in the paper we did not really use them since we designed a different reward. But for now, since the implementation needs them for creating zeros matrix of the same size, please download them from [here](https://drive.google.com/file/d/1iQww68_w9V29dl3rwFdBpk3sSU2z1UFd/view?usp=sharing) and put the file where you set `config_train.tfrecord_data_dir` .
I will modify the code later, so there is no need for these files.

**Comment by shaoniana1997** [comment]
Thank you for your timely reply.I will try it .I want to ask you another question,when i  bash run_emorl.sh with comet,
the train epoch is 5 , But I haven't finished training an epoch in two days. Is this normal？？
And my device is GTX2080Ti.
